frontend/src/Routes/DynamicRoute/DynamicRoute.tsx

```diff
@@ -2,8 +2,10 @@ import * as React from 'react';
 import { Route, Routes } from 'react-router-dom';
 import { useExtensions, useResolvedExtensions } from '@openshift/dynamic-plugin-sdk';
 import { isRoutePage as isDynamicRoutePage, RoutePage as DynamicRoutePage, isContextProvider } from '@openshift/dynamic-plugin-sdk-extensions';
-import { Bullseye, Spinner, PageSection } from '@patternfly/react-core';
-import { ErrorState } from '@redhat-cloud-services/frontend-components/ErrorState';
+import { Bullseye } from '@patternfly/react-core/dist/dynamic/layouts/Bullseye';
```

Comment:
Again I'd check out https://github.com/patternfly/patternfly-react/pull/9298

Reply:
I think this is coming from forbid-pf-relative-imports rule in @redhat-cloud-services/eslint-config-redhat-cloud-services.